platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapGestureDetector.java

```diff
@@ -776,6 +814,41 @@ public boolean onMultiFingerTap(@NonNull MultiFingerTapGestureDetector detector,
     }
   }
 
+  private Animator createPitchAnimator(double currentPitch, double pitchAddition, long animationTime) {
```

Comment:
This animator seems unnecessary to me. I've used `transform.setTilt(transform.getTilt() +/- 1);` instead for each key event and it worked just fine and was consistent with the move event.

Reply:
Removed the animator and have switched to `transform.getTilt() + 1` and ``transform.getTilt() - 1` for respective tilting. Both work.

Reply:
Is the PR ready for a review? (seeing that it's still a draft)

Reply:
Nope, not ready @LukasPaczos